bin/pycbc_optimize_snr

```diff
@@ -237,67 +326,46 @@ sample_rate = fp['sample_rate'][()]
 
 extra_args = [data, coinc_times, coinc_ifos, flen,
               approximant, flow, f_end, delta_f, sample_rate]
-extra_kwargs = {}
-extra_kwargs['args'] = extra_args
-
-mass1 = fp['mass1'][()]
-mass2 = fp['mass2'][()]
-spin1z = fp['spin1z'][()]
-spin2z = fp['spin2z'][()]
-mchirp = mchirp_from_mass1_mass2(mass1, mass2)
+
+mchirp = mchirp_from_mass1_mass2(
+    fp['mass1'][()],
+    fp['mass2'][()]
+)
 minchirp = mchirp * (1 - mchirp / 50.0)
 maxchirp = mchirp * (1 + mchirp / 50.0)
```

Comment:
It looks like there's a bug here. I think this was supposed to be *fractional* bounds on mchirp (maybe ones that increase with the mchirp??). But as a specific fail case: if the mchirp is 50, this will search down to a mchirp of 0 and then things will break!

Reply:
Although I guess the lines below catch that, but I still think the bounds aren't ideal!

Reply:
Ok let's keep this in mind, but solve it as a separate issue.